tests/test_directory_sync.py

```diff
@@ -8,7 +8,7 @@
 from workos.utils.request import RESPONSE_TYPE_CODE
 
 
-class TestSSO(object):
+class TestDirectorySync(object):
```

Comment:
Small unrelated change. I noticed this typo while I was copying other people's code :)